templates/etc/haproxy/haproxy.cfg.j2

```diff
@@ -6,6 +6,8 @@ global
 defaults
 {% include 'defaults.cfg.j2' %}
 
+{% include 'peers.cfg.j2' %}
```

Comment:
I can't find anything that conclusively proves it's a global - other than http://cbonte.github.io/haproxy-dconv/configuration-1.5.html#3.5 listing it under "globals" heading.   All the examples I've found couple it with the backends - but I've now moved it to the globals section.